merp_custom_logotype/tests/test_checking_logotype.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+
+from odoo.tests.common import TransactionCase
+from odoo.exceptions import Warning
+
+
+class TestCheckingLogotype(TransactionCase):
+
+    def test_check_type_logo(self):
+        user = self.env['res.users'].create({
+            'name': 'admin',
+            'login': 'login@email.com',
+            'sel_groups_42_43_44': 44,
```

Comment:
WHat is this? Groups are not added to user like this 
Read here https://www.odoo.com/forum/help-1/question/add-users-to-group-through-python-code-78286